fdbclient/NativeAPI.actor.cpp

```diff
@@ -2192,6 +2217,137 @@ ACTOR Future<Void> watchValue(Future<Version> version, Key key, Optional<Value>
 	}
 }
 
+ACTOR Future<Void> watchStorageServerResp(Key key, Database cx) {
+	loop {
+		try {
+			state Reference<WatchMetadata> metadata = cx->getWatchMetadata(key);
+			if (!metadata.isValid()) {
+				return Void();
+			}
+
+			state Future<Version> watchFutureSS = watchValue(Future<Version>(metadata->version), key, metadata->value, cx, metadata->info, metadata->tags);
+			Version watchVersion = wait(watchFutureSS);
```

Comment:
Since you aren't using `watchFutureSS` except to wait on the version, there's no need to make it a variable. Something like `Version watchVersion = wait(watchValue(...))` is cleaner.